core/embed/rust/src/ui/model_mercury/component/button.rs

```diff
@@ -413,11 +413,13 @@ pub struct ButtonStyleSheet {
     pub disabled: &'static ButtonStyle,
 }
 
+// TODO: radius and borders not needed in T3T1 - sync with BL
```

Comment:
I think this should not break anything and will look more consistent BL<->FW, let's do it.

Reply:
https://github.com/trezor/trezor-firmware/pull/3740/commits/944a2ee6e61fd7cba9b07299672c6aabcfbdec5e